sycl/include/CL/sycl/ONEAPI/matrix/matrix-amx.hpp

```diff
@@ -196,51 +197,55 @@ struct joint_matrix<
   joint_matrix(Group sg) {}
 };
 
-} // namespace matrix
+} // namespace experimental::matrix
 
 namespace detail {
 
 template <typename Group, typename T, size_t NumRows, size_t NumCols,
-          matrix::matrix_layout Layout>
-inline __SYCL_ALWAYS_INLINE static
-    typename std::enable_if<(NumRows > matrix::tile_size) ||
-                                (NumCols * sizeof(T) / 4 > matrix::tile_size),
-                            void>::type
-    submatrix_load(detail::submatrix<T> &sub_m,
-                   matrix::joint_matrix<Group, T, NumRows, NumCols, Layout> jm,
-                   uint32_t row, uint32_t col, size_t stride,
-                   matrix::matrix_layout layout, bool shouldreload) {
+          experimental::matrix::matrix_layout Layout>
+inline __SYCL_ALWAYS_INLINE static typename std::enable_if<
+    (NumRows > experimental::matrix::tile_size) ||
+        (NumCols * sizeof(T) / 4 > experimental::matrix::tile_size),
+    void>::type
```

Comment:
I think, from now on we require C++17, so we can use some of its advantages.
```suggestion
inline __SYCL_ALWAYS_INLINE static typename std::enable_if_t<
    (NumRows > experimental::matrix::tile_size) ||
        (NumCols * sizeof(T) / 4 > experimental::matrix::tile_size),
    void>
```